packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -454,38 +456,59 @@ export const prepareAuctionBook = (baggage, zcf, makeRecorderKit) => {
           });
           return state.bookDataKit.recorder.write(bookData);
         },
-        observeQuoteNotifier() {
+        // Ensure that there is an observer monitoring the quoteNotifier. We
+        // assume that all failure modes for quoteNotifier eventually lead to
+        // fail or finish.
+        ensureQuoteNotifierObserved() {
           const { state, facets } = this;
           const { collateralBrand, bidBrand, priceAuthority } = state;
 
+          if (observedBrands.has(collateralBrand)) {
```

Comment:
Why are we using the `collateralBrand` as key instead of the identity of this auctionBook? Are we guaranteed there is only ever a 1:1 relationship between auctionBook and collateralBrand?

Reply:
The rest of `auctionBook` and `auctioneer` use the `collateralBrand`, so we're already reliant on it being 1:1.

Reply:
So there is never multiple bid brands for a given collateral? I'm just wary of assumptions like this that aren't super clear.

Reply:
All of the bids are in IST, so there's no chance of that.

With Vaults, we explicitly intended that we might want to have multiple vaultManagers for a given currency that could have different minimums, or other requirements, and charge different rates.

With auctions, it's a mistake to split the bidding equity across multiple auctions, so the design was always that each auctionBook would handle all the bids and asks for a particular asset.

Reply:
Yeah I think having a single bidding denom is the assumption that wasn't clear to me.